core/src/providers/anthropic.rs

```diff
@@ -82,22 +82,26 @@ impl AnthropicLLM {
             .iter()
             .map(|cm| -> String {
                 format!(
-                    "\n\n{}: {}",
+                    "\n\n{}: {}{}",
                     match cm.role {
-                        ChatMessageRole::System => "System",
+                        ChatMessageRole::System => "Human",
```

Comment:
So basically we use Human for everything that is not Claude-generated and Asssitant for Claude only ?

Reply:
As per their request